resources/brave-youtube-quality.js

```diff
@@ -0,0 +1,56 @@
+/// brave-youtube-quality.js
+(function() {
+    const IS_MOBILE_YOUTUBE = window.location.hostname == 'm.youtube.com';

```

Comment:
The hostname could be `www.youtube.com`

Reply:
I think we need to include both as we need m.youtube.com for mobile sites and www.youtube.com for desktop sites option